cafebabel/static/js/script.js

```diff
@@ -56,3 +56,56 @@ Array.from(document.querySelectorAll('.authentication-form > div > input')).forE
         parent.classList.remove('active', 'completed')
     })
 })
+
+/* highlight file upload area on hover or dragenter */
+function addListenerMulti(element, events, fn) {
+    events.split(' ').forEach(event => element.addEventListener(event, fn, false)) /* https://stackoverflow.com/questions/8796988/binding-multiple-events-to-a-listener-without-jquery */
+}
+const fileInput = document.querySelector('.file input')
+const dropArea = document.querySelector('canvas')
+if (fileInput) {
+    addListenerMulti(fileInput, 'dragenter focus click', () => {
+        dropArea.classList.add('active')
+    })
+    addListenerMulti(fileInput, 'dragleave blur drop', () => {
+        dropArea.classList.remove('active')
+    })
+    fileInput.addEventListener('change', () => {
+        document.querySelector('.file label').innerHTML(' ')
```

Comment:
Why removing the label when the file has changed?

Reply:
Im not done with that, I would like to display the file name as soon as it's dropped (or chosen). I was thinking of displaying it replacing the label content. But maybe not the best from an accessibility point of view ?